.github/workflows/codeql-analysis.yml

```diff
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master, 0.5.0, 0.5.1, 0.5.2, release-* ]
+    branches: [ main, 0.5.0, 0.5.1, 0.5.2, release-* ]
```

Comment:
Can't we remove `0.5.0, 0.5.1, 0.5.2`?

Reply:
At present, I think these branch could be removed, but this PR only aims to update the main branch, let's do the cleanup work not limited to this place in other PR.